Makefile

```diff
@@ -0,0 +1,86 @@
+# Copyright Concurrent Technologies Corporation 2021
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# Makefile -  execution wrapper for common fapolicy-analyzer development tasks
+
+list:
+	@echo
+	@echo "  Usage: make [target]"
+	@echo
+	@echo "       fapolicy-analyzer - High level common operation targets"
+	@echo
+	@echo "     list    - Display common development targets"
+	@echo "     dev-shell  - Install deps, build bindings, start venv shell"
+	@echo "     run     - Execute the fapolicy-analyzer"
+	@echo "     listall - Display all targets"
+	@echo "    "
+
+###############################################################################
+# Development environment population and creation
+dev-shell: pipenv-install pybindings
+	@echo
+	@echo "--- Starting a fapolicy-analyzer development shell"
+	pipenv shell
+	@echo "--- Development shell terminated"
+
+dev-env: pipenv-install pybindings
+
+pipenv-install:
+	@echo "  |--- Installing pipenv dependencies..."
+	pipenv install --dev
+
+pybindings: pipenv-install
+	@echo "  |--- Generating python bindings..."
+	pipenv run python setup.py develop
+
+###############################################################################
+# fapolicy-analyzer execution
+run: dev-env
+	pipenv run python -m fapolicy_analyzer.ui
+
+run-verbose: dev-env
```

Comment:
Don't use a separate target here, just allow an env var to be set.

A common `make` pattern found in the wild looks like

`V=1 make build`

Reply:
This was first pass. Things are still being worked. I'd prefer to pass and handle a CL arg. but I get your point.

Reply:
Added the ability to pass in CL arguments via the OPTIONS environment variable when using the 'run' target, e.g.
`$ OPTIONS='-v -h' make run`